client/src/panels/panel_declarations/people/employee_age.js

```diff
@@ -166,31 +166,27 @@ export const declare_employee_age_panel = () =>
               <TM k={level + "_employee_age_text"} args={info} />
             </Col>
             <Col size={12} isGraph extraClasses="zero-padding">
-              <div aria-hidden="true">
-                <TabbedContent
-                  tab_keys={["age_group", "avgage"]}
-                  tab_labels={{
-                    age_group: text_maker("age_group"),
-                    avgage: text_maker("avgage"),
-                  }}
-                  tab_pane_contents={{
-                    age_group: (
-                      <div id={"emp_age_tab_pane"}>
-                        <GraphOverlay>
```

Comment:
Import can be removed

Reply:
Oh I think removing this was a mistake from rebase-ing

Reply:
..Or else I can't remember why I removed GraphOverlay haha